client/sections.js

```diff
@@ -218,7 +218,7 @@ const sections = [
 	{
 		name: 'hosting-overview',
 		paths: [ '/hosting-overview' ],
-		module: 'calypso/hosting-overview',
+		module: 'calypso/hosting',
```

Comment:
I got an error when accessing `/hosting-overview` or `/hosting-config`

```
cannot find the chunk "hosting"
```

Reply:
Oops forgot to revert this part too. Thanks.